### ISSUE TITLE
BatteryLevelReceiver ClassNotFoundException

### ISSUE THREAD

**Issue by Petr-Kubista** [clostra/newnode#117]
We're having tons of crash reports coming from our latest Android build using NewNode lib version 2.0.0.

```
Fatal Exception: java.lang.RuntimeException: Unable to instantiate receiver com.clostra.newnode.internal.NearbyHelper.BatteryLevelReceiver: java.lang.ClassNotFoundException: Didn't find class "com.clostra.newnode.internal.NearbyHelper.BatteryLevelReceiver" on path: DexPathList[[zip file "/data/app/gov.bbg.voa-2/base.apk", zip file "/data/app/gov.bbg.voa-2/split_config.armeabi_v7a.apk", zip file "/data/app/gov.bbg.voa-2/split_config.xxxhdpi.apk"],nativeLibraryDirectories=[/data/app/gov.bbg.voa-2/lib/arm, /data/app/gov.bbg.voa-2/base.apk!/lib/armeabi-v7a, /data/app/gov.bbg.voa-2/split_config.armeabi_v7a.apk!/lib/armeabi-v7a, /data/app/gov.bbg.voa-2/split_config.xxxhdpi.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib]]
       at android.app.ActivityThread.handleReceiver(ActivityThread.java)
       at android.app.ActivityThread.-wrap20(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java)
       at android.os.Handler.dispatchMessage(Handler.java)
       at android.os.Looper.loop(Looper.java)
       at android.app.ActivityThread.main(ActivityThread.java)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java)

Caused by java.lang.ClassNotFoundException: Didn't find class "com.clostra.newnode.internal.NearbyHelper.BatteryLevelReceiver" on path: DexPathList[[zip file "/data/app/gov.bbg.voa-2/base.apk", zip file "/data/app/gov.bbg.voa-2/split_config.armeabi_v7a.apk", zip file "/data/app/gov.bbg.voa-2/split_config.xxxhdpi.apk"],nativeLibraryDirectories=[/data/app/gov.bbg.voa-2/lib/arm, /data/app/gov.bbg.voa-2/base.apk!/lib/armeabi-v7a, /data/app/gov.bbg.voa-2/split_config.armeabi_v7a.apk!/lib/armeabi-v7a, /data/app/gov.bbg.voa-2/split_config.xxxhdpi.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib]]
       at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
       at android.app.ActivityThread.handleReceiver(ActivityThread.java)
       at android.app.ActivityThread.-wrap20(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java)
       at android.os.Handler.dispatchMessage(Handler.java)
       at android.os.Looper.loop(Looper.java)
       at android.app.ActivityThread.main(ActivityThread.java)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java)
```

These crashes happen on Samsung (e.g. Galaxy Note Fan Edition, Galaxy Note5, Galaxy Tab S2, Galaxy A8) and LG (e.g. LG K9, G6, G Pad IV) devices running Android 7.

We're switching nearby discovery off prior to initializing:
`NewNode.setRequestDiscoveryPermission(false);`

**Comment by ghazel** [comment]
This was fixed in https://github.com/clostra/newnode/commit/98a17628483c069d5bdaeccd58009eaa29075500 however there is not a new library release yet.

**Comment by ghazel** [comment]
Released in 2.0.3
